vortex_allocator/src/allocator_ros.cpp

```diff
@@ -8,16 +9,20 @@
 #include "vortex/eigen_typedefs.h"
 #include "vortex/eigen_helper.h"
 
-Allocator::Allocator(ros::NodeHandle nh) : nh(nh)
+Allocator::Allocator(ros::NodeHandle nh)
+ :
```

Comment:
Roslint: 
`vortex_allocator/src/allocator_ros.cpp:13:  Weird number of spaces at line-start.  Are you using a 2-space indent?  [whitespace/indent] [3]`